frontend/src/helpers/API.js

```diff
@@ -62,15 +62,15 @@ const signOut = (setLoginOutError) => {
       },
     })
     .then((response) => {
-      const [status, data] = [response.data.status, response.data];
+      const [status, data] = [response.status, response.data];
 
-      if (status === 200 || (status === 400 && data.error === 'Invalid.')) {
+      if (status === 200 || (status === 401 && data.error === 'Invalid.')) {
```

Comment:
Do we need a message on an error on sign out?  
This case actually seems like it might be an internal error.

Reply:
Looking at the back end, I think the `Invalid` message only occurs when the token is not found. So, it should definitely be true they are not logged in with that token.

However, we should make this is true.